rules/plugin-unicorn.js

```diff
@@ -27,15 +30,20 @@ module.exports = {
 		'unicorn/no-zero-fractions': 'off',
 		'unicorn/number-literal-case': 'off',
 		'unicorn/prefer-add-event-listener': 'error',
+		'unicorn/prefer-dataset': 'off',
```

Comment:
```suggestion
		'unicorn/prefer-dataset': 'error',
```
https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-dataset.md
^ kinda like the idea of this, if we ever do have to touch the dom, which we don't but if we do this is _tidy_

Reply:
I wasn't sure what to do with this rule since I don't do a whole lot on the front end. I'm all for enabling it and then disabling it laser if needed.